orchestra/static/orchestra/project-management/project-management.html

```diff
@@ -6,16 +6,38 @@
     <div class="container-fluid">
       <div class="row padded">
         <div class="col-lg-12 col-md-12 col-sm-12">
-          <ui-select class="project-description" ng-model="vis.dataService.currentProject"
-              ng-change="vis.dataService.setSelectedProject()"
-              ng-disabled="vis.dataService.loading">
-            <ui-select-match>
-              <span ng-bind="projectDescription($select.selected)"></span>
-            </ui-select-match>
-            <ui-select-choices repeat="item in (vis.dataService.allProjects | toArray | filter: $select.search) track by item.id">
-              <span ng-bind="projectDescription(item)"></span>
-            </ui-select-choices>
-          </ui-select>
+          <div>
```

Comment:
Looking at the interface in the demo, I like the button where it was. right now the heights of the search box and the button are different, and the color of the button is funky. i'd make the button have the same style as the other buttons on its row.